common/courses/participants.ts

```diff
@@ -268,7 +268,7 @@ export async function leaveSubcourse(subcourse: Subcourse, pupil: Pupil) {
     logger.info(`Pupil(${pupil.id}) left Subcourse(${subcourse.id})`);
     await logTransaction('participantLeftCourse', pupil, { subcourseID: subcourse.id });
 
-    const course = prisma.course.findUnique({ where: { id: subcourse.courseId } });
+    const course = await prisma.course.findUnique({ where: { id: subcourse.courseId } });
```

Comment:
We always find new places like this 😞

Reply:
jup, need better linters. It's so hard to spot these in code review ...